src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -807,6 +838,26 @@ protected virtual bool PopulateXmlDocument(XmlDocument xmlDoc)
                     note.SetAttribute("y", n.Y.ToString(CultureInfo.InvariantCulture));
                 }
 
+                //save the annotation
+                var annotationList = xmlDoc.CreateElement("Annotations");
+                root.AppendChild(annotationList);
+                foreach (var n in annotations)
+                {
+                    var annotation = xmlDoc.CreateElement(n.GetType().ToString());
+                    annotationList.AppendChild(annotation);
+                    annotation.SetAttribute("guid", n.GUID.ToString());
+                    annotation.SetAttribute("text", n.AnnotationText);                   
+                    annotation.SetAttribute("left", n.Left.ToString(CultureInfo.InvariantCulture));
+                    annotation.SetAttribute("top", n.Top.ToString(CultureInfo.InvariantCulture));
+                    annotation.SetAttribute("width", n.Width.ToString(CultureInfo.InvariantCulture));
+                    annotation.SetAttribute("height", n.Height.ToString(CultureInfo.InvariantCulture));                   
+                    annotation.SetAttribute("annotationColor", (n.BackGroundColor == null ? "" : n.BackGroundColor.ToString()));
+                    annotation.SetAttribute("fontSize", n.FontSize.ToString(CultureInfo.InvariantCulture));
+                    annotation.SetAttribute("initialTop", n.InitialTop.ToString(CultureInfo.InvariantCulture));
+                    annotation.SetAttribute("initialHeight", n.InitialHeight.ToString(CultureInfo.InvariantCulture));
+                    annotation.SetAttribute("ModelGUIDs", string.Join(",", n.SelectedModels.Select(z => z.GUID)));
+                }
```

Comment:
Let's have this code live in `AnnotationModel`.  The pattern here with `Connection` and `Note` is dubious at best.  :/